tgui/packages/common/uuid.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file
+ * @copyright 2020 Aleksej Komarov
+ * @license MIT
+ */
+ 
+/**
+ * UUID v4 string generator
+ *
+ * @return {string}
+ */
+export const CreateUuid = function () {
+  let d = new Date().getTime();
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, c => {
+    const r = (d + Math.random() * 16) % 16 | 0;
+    d = Math.floor(d / 16);
+    return (c === 'x' ? r : (r & 0x3 | 0x8)).toString(16);
+  });
+};
```

Comment:
Verbatim replace this file with the suggestion, must be lower case, and missing newline at the end of file. (Have you installed the `editorconfig` vscode extension?)

```suggestion
/**
 * Creates a UUID v4 string
 *
 * @return {string}
 */
export const createUuid = () => {
  let d = new Date().getTime();
  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, c => {
    const r = (d + Math.random() * 16) % 16 | 0;
    d = Math.floor(d / 16);
    return (c === 'x' ? r : (r & 0x3 | 0x8)).toString(16);
  });
};
```